rpc/server.go

```diff
@@ -71,7 +71,7 @@ type Server struct {
 //   - batchConcurrency: maximum number of goroutines processing batch requests (0 = unlimited)
 //   - traceRequests: enables detailed request tracing in logs
 //   - debugSingleRequest: enables debugging of individual requests
-//   - disableStreaming: disables streaming responses
+//   - disableStreaming: when true, forces immediate response for subscription requests
```

Comment:
not really true: "streaming of large json responses" (usually it's large array of objects) - does send to client parts of response when they are ready (items of array). Means client can start parsing and handling results in lazy way. Problem is: some block's traces on EthMainnet are 80Gb.